src/main/java/at/hannibal2/skyhanni/config/features/inventory/AuctionHousePriceComparisonConfig.java

```diff
@@ -12,8 +12,8 @@ public class AuctionHousePriceComparisonConfig {
     @Expose
     @ConfigOption(
         name = "Show Price Comparison",
-        desc = "Highlight auctions based on the difference between their estimated value and the value they are listed for. §eCan " +
-            "be very inaccurate."
+        desc = "Highlight auctions based on the difference between their estimated value and the value they are listed for. " +
+            "§eCan be very inaccurate."
```

Comment:
missing line break? or not good because too long text above?

Reply:
in this case, seems like the line is originally too short for a line break, but lengthening it in this manner should fix that. would you prefer lengthening it or keeping it the way it was?

Reply:
the new line is
```diff
+        desc = "Highlight auctions based on the difference between their estimated value and the value they are listed for.\n" +
+            "§eThis may be very inaccurate at times and only provides an estimate."
```

Reply:
It's really small now. But i think its fine.
![grafik](https://github.com/hannibal002/SkyHanni/assets/24389977/fd808205-fb6a-46c1-bd39-0339e41efc8d)

Reply:
👍